docs/src/includes/_types.md

```diff
@@ -3307,30 +3307,30 @@ Type | Name | Description
 
 Type | Name | Description
 ---- | ---- | -----------
-[EntityDB](#EntityDB) | [type](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=type) | 
+[EntityDB](#EntityDB) | [type](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=type) | Type of the entity, contains special properties etc. If you want to edit them just for this entity look at the [EntityDB](#EntityDB)
 [Entity](#Entity) | [overlay](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=overlay) | 
-int | [flags](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=flags) | 
-int | [more_flags](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=more_flags) | 
-int | [uid](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=uid) | 
-int | [animation_frame](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=animation_frame) | 
-int | [draw_depth](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=draw_depth) | Don't edit this directly, use `set_draw_depth`
-float | [x](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=x) | 
-float | [y](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=y) | 
-int | [layer](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=layer) | 
-float | [width](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=width) | 
-float | [height](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=height) | 
-float | [special_offsetx](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=special_offsetx) | 
-float | [special_offsety](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=special_offsety) | 
-float | [tile_width](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=tile_width) | 
-float | [tile_height](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=tile_height) | 
+int | [flags](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=flags) | see [flags.hpp](https://github.com/spelunky-fyi/overlunky/blob/main/src/game_api/flags.hpp) entity_flags
```

Comment:
Should probably set the type of these to ENT_FLAG etc, that's more useful than flags.hpp.

Reply:
Sure, but I'll leave the comment